src/components/Drawer.jsx

```diff
@@ -102,10 +98,16 @@ const CloseButton = ({ onClose }) => {
 };
 
 Drawer.propTypes = {
-    isOpen: PropTypes.bool,
+    isOpen: PropTypes.bool.isRequired,
     title: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
-    content: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
+    content: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
```

Comment:
why is this change required ?

Reply:
Cos I don't think that this component'll be used without children

Reply:
Agreed, Dialog should always have component